Dockerfile

```diff
@@ -7,15 +7,74 @@ LABEL "com.github.actions.description"="Generates terraform code that calls terr
 LABEL "com.github.actions.icon"="activity"
 LABEL "com.github.actions.color"="blue"
 
+# Install the cloudposse alpine repository
 ADD https://apk.cloudposse.com/ops@cloudposse.com.rsa.pub /etc/apk/keys/
-RUN echo "@cloudposse https://apk.cloudposse.com/3.11/vendor" >> /etc/apk/repositories
+RUN echo "https://apk.cloudposse.com/3.12/vendor" >> /etc/apk/repositories
+RUN echo "https://alpine.global.ssl.fastly.net/alpine/edge/testing" >> /etc/apk/repositories
+RUN echo "https://alpine.global.ssl.fastly.net/alpine/edge/community" >> /etc/apk/repositories
 
-RUN apk add -u \
-				git \
-				variant2@cloudposse \
-				go-jsonnet@cloudposse==0.17.0-r0
```

Comment:
I couldn't get these to install anymore. I was using the repo that redirects for your apk.cloudposse.com cname.
specifically I nabbed this boi https://github.com/cloudposse/packages/blob/master/apk/Dockerfile-3.12